platform/mbed-trace/mbed_lib.json

```diff
@@ -19,6 +19,12 @@
             "value": "malloc",
             "macro_name": "MEM_ALLOC"
         },
+        "color-theme": {
+            "help": "Set color theme. NULL for bright, 1 for dark.",
```

Comment:
Why the choice of "null" and "1"? Why not "0" and "1"?

```diff
- "help": "Set color theme. NULL for bright, 1 for dark.",
+ "help": "Set color theme. 0 for readable, 1 for unobtrusive.",
```

Reply:
I was just copying it from a different place. 0/1 sounds good as well.

Reply:
Fixed.